<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeamFoundation::Internal::SettingsPage</class>
 <widget class="QWidget" name="TeamFoundation::Internal::SettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>359</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="generalGroupBox">
     <property name="title">
      <string>Configuration</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="commandLabel">
        <property name="text">
         <string>Team Foundation command (tf.exe):</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="Utils::PathChooser" name="pathChooser" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="tfptLabel">
        <property name="text">
         <string>Team Foundation Power Tools command (tfpt.exe):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="Utils::PathChooser" name="tfptPathChooser" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="userGroupBox">
     <property name="title">
      <string>Authentication</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="usernameLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="passwordLineEdit">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="miscGroupBox">
     <property name="title">
      <string>Miscellaneous</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="timeOutLabel">
        <property name="text">
         <string>Timeout:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="timeOutSpinBox">
        <property name="suffix">
         <string>s</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>127</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
